modules/op-about-triggers.adoc

```diff
@@ -90,13 +90,15 @@ spec:
 <1> The API version of the `TriggerTemplate` resource. In this example, `v1alpha1`.
 <2> Specifies the type of Kubernetes object. In this example, `TriggerTemplate`.
 <3> Unique name to identify the `TriggerTemplate` resource.
-<4> Parameters supplied by the `TriggerBinding` or `EventListerner` resources.
+<4> Parameters supplied by the `TriggerBinding` and `EventListerner` resources.
```

Comment:
It should just be `Parameters supplied by the 'TriggerBindig' resource not `Parameters supplied by the 'TriggerBinding' and 'EventListener' resources`
cc: @savitaashture

Reply:
Done.